app/lib/features/space/widgets/space_nav_bar.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:acter/common/utils/routes.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+enum SpaceNavItem { overview, pins, events, chats, spaces, members }
+
+class SpaceNavBar extends StatefulWidget {
+  final String spaceId;
+  final int selectedIndex;
+
+  const SpaceNavBar({
+    super.key,
+    required this.selectedIndex,
+    required this.spaceId,
+  });
+
+  @override
+  State<SpaceNavBar> createState() => _SpaceNavBarState();
+}
+
+class _SpaceNavBarState extends State<SpaceNavBar>
+    with TickerProviderStateMixin {
+  late TabController _tabController;
+
+  @override
+  void initState() {
+    _tabController = TabController(
+      length: 6,
+      vsync: this,
+      initialIndex: widget.selectedIndex,
+      animationDuration: const Duration(milliseconds: 500),
+    );
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        final useCols = constraints.maxWidth < (150 * 6);
+        int minItemWidth = useCols ? 90 : 150;
+        final minTotalWidth = minItemWidth * 6;
+        bool scrollBar = false;
+        if (minTotalWidth > constraints.maxWidth) {
+          scrollBar = true;
+        }
+        return Container(
+          width: MediaQuery.of(context).size.width,
+          decoration: BoxDecoration(
+            borderRadius: const BorderRadius.only(
+              topLeft: Radius.circular(10),
+              topRight: Radius.circular(10),
+            ),
+            color: Theme.of(context).colorScheme.onPrimary,
+          ),
+          margin: const EdgeInsets.symmetric(horizontal: 20, vertical: 10),
+          child: TabBar(
+            controller: _tabController,
+            isScrollable: scrollBar,
+            labelColor: Colors.white,
+            labelStyle: useCols
+                ? Theme.of(context).textTheme.labelSmall
+                : Theme.of(context).textTheme.bodySmall,
+            indicatorColor: Theme.of(context).colorScheme.tertiary,
+            padding: EdgeInsets.zero,
+            tabs: [
+              tabViewItemUI(
+                title: 'Overview',
+                iconData: Atlas.layout_half_thin,
+                onTap: () => context.goNamed(
+                  Routes.space.name,
+                  pathParameters: {'spaceId': widget.spaceId},
+                ),
+              ),
+              tabViewItemUI(
```

Comment:
as said on the file-comment, whether they should be shown or not is depending on the space-state. That's already implemented in the existing top-navbar-widget via providers - either recreate the existing code or reuse the existing widget (I'd recommend to do the latter).